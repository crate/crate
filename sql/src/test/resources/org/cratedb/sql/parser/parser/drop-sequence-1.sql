DROP SEQUENCE test.bad_sequence RESTRICT
