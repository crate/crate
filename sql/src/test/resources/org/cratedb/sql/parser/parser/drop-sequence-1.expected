org.cratedb.sql.parser.parser.DropSequenceNode@16bdb503
name: test.bad_sequence
statementType: DROP SEQUENCE bad_sequence
dropBehavior: 1
existenceCheck: NO_CONDITION